abci/example/kvstore/kvstore.go

```diff
@@ -85,8 +116,20 @@ func (app *Application) Info(req types.RequestInfo) (resInfo types.ResponseInfo)
 	}
 }
 
-// tx is either "key=value" or just arbitrary bytes
-func (app *Application) HandleTx(tx []byte) *types.ResponseDeliverTx {
+// tx is either "val:pubkey!power" or "key=value" or just arbitrary bytes
```

Comment:
Is the val: behavior a new addition? (No objection, just wasn't sure where it came from)

Edit: Oh, huh, TIL.